archlinuxcn/llvm-git/PKGBUILD

```diff
@@ -53,9 +58,7 @@ prepare() {
     # llvm-project contains a lot of stuff, remove parts that aren't used by this package
     rm -rf "$srcdir"/fakeinstall
 
-  cd llvm/projects
-  git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git
-
+  cp -r $srcdir/SPIRV-LLVM-Translator llvm/projects
 }
 
 build() {
```

Comment:
```
==> ERROR: A failure occurred in build().
```
https://fars.ee/Nclk.log

Reply:
the issue is that there isn't enough storage on your device.

Reply:
"/build/llvm-git/src/_build/bin/llvm-exegesis" to
  "/build/llvm-git/src/fakeinstall/usr/bin/llvm-exegesis": No space left on
  device.

Reply:
I'm using a build server provided by the Arch Linux CN community, so insufficient storage space is unlikely.

Reply:
so the size of the build server needs to be expanded

Reply:
@Zenvie you ran out the tmpfs space for the chroot by default. lilac has been instructed to `-r` to choose another place for it.

Reply:
@lilydjwg Is there an option in lilac.yaml that can instruct lilac to build the package outside tmpfs?

Reply:
> @Zenvie you ran out the tmpfs space for the chroot by default. lilac has been instructed to use `-r` to choose another place for it.

This parameter has the unintended consequence of creating files that I lack the necessary permissions to delete.

Reply:
@oldherl it's in [lilac.py](https://github.com/archlinuxcn/lilac/blob/master/schema-docs/lilac-py-fields.md).

@Zenvie yes....